packages/sveltekit/src/executors/sveltekit/executor.ts

```diff
@@ -9,17 +28,39 @@ export default async function runExecutor(
   const projectDir = context.workspace.projects[context.projectName].root;
   const projectRoot = joinPathFragments(`${context.root}/${projectDir}`);
   const portOption = options.command === 'dev' ? ` --port ${options.port}` : '';
+  const projectOutput = joinPathFragments(context.root, 'dist/', projectDir);
 
   const result = await runCommands(
     {
-      command: `svelte-kit ${options.command}${portOption}`,
       cwd: projectRoot,
+      commands: [
+        {
+          // creates folder if it doesn't exist
+          // otherwhise cp command doesn't work
+          command: `mkdir -p ${projectOutput}`,
```

Comment:
I'd do this kind of stuff in code before run the sveltekit command instead of just spawning cli tools. I'n the long run I'll add this for the cli call as well into code